src/bp/core/config/config-loader.ts

```diff
@@ -105,7 +105,7 @@ export class ConfigProvider {
   }
 
   public async getModulesListConfig() {
-    const enabledByDefault = ['analytics', 'basic-skills', 'builtin', 'channel-web', 'nlu', 'qna']
+    const enabledByDefault = ['analytics', 'basic-skills', 'builtin', 'channel-web', 'nlu', 'qna', 'extensions']
```

Comment:
I'm not sure this module should be enabled by default. By design extensions are extensions which you want to enable only if you need them. An extension enabled by default becomes a builtin thing ...

Reply:
I'd be tempted to agree with you, but since most users won't bother playing around with the configuration and enable other modules, it makes sense to enable them by default, until we get a user-friendly modules list where a user can quickly enable/disable them and see which one are available

Reply:
alright then, agreed !